stetho/src/main/java/com/facebook/stetho/inspector/elements/AbstractChainedDescriptor.java

```diff
@@ -34,11 +34,13 @@
  * @param <E> the class that this descriptor will be describing for {@link DocumentProvider},
  * {@link Document}, and ultimately {@link DOM}.
  */
-public abstract class AbstractChainedDescriptor<E> extends Descriptor implements ChainedDescriptor {
+public abstract class AbstractChainedDescriptor<E>
+    extends Descriptor<E> implements ChainedDescriptor {
 
-  private Descriptor mSuper;
+  private Descriptor<Object> mSuper;
```

Comment:
see my comments on `ChainedDescriptor` ... I'd love it if we can make this `Descriptor<? super E>`